SingularityUI/app/views/TaskPoller.coffee

```diff
@@ -0,0 +1,142 @@
+HistoricalTasks = require '../collections/HistoricalTasks'
+TaskHistoryItem = require '../models/TaskHistoryItem'
+
+TaskFiles = require '../collections/TaskFiles'
+RequestTasks = require '../collections/RequestTasks'
+
+taskPollerWaitingTemplate = require 'templates/vex/taskPollerWaiting'
+taskPollingFailureTemplate = require 'templates/vex/taskPollingFailure'
+
+Utils = require '../utils'
+
+vex = require 'vex.dialog'
+
+interval = (a, b) -> setInterval(b, a)  # f u javascript
+timeout = (a, b) -> setTimeout(b, a)
+
+TIMEOUT_MINUTES = 1 # Modify this
```

Comment:
Might be easier to get this to millis using something like `moment.duration(1, 'minutes').asMilliseconds()`, and gets rid of the need for using comments like 'Don't modify this'